unit/inettest.c

```diff
@@ -59,7 +61,7 @@ static ssize_t do_request4(const knot_msg *kmsg, size_t len, knot_msg *kresp)
 	memset(&addr,0,sizeof(addr));
 	addr.sin_family = AF_INET;
 	addr.sin_addr.s_addr = htonl(INADDR_ANY);
-	addr.sin_port = htons(9994);
+	addr.sin_port = htons(8884);
```

Comment:
These numbers are the ports that are running the inetbrd.
Actually, these files (inettest and ktest) are not representing a unit test but an integration test in my opinion, because in unit tests we don't need to run the service. In unit tests, we should mock the service and test just the functionality.